.asf.yaml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+github:
+  description: "Apache NiFi - MiNiFi C++"
+  homepage: https://nifi.apache.org/minifi/
+  labels:
+    - nifi
+    - c++
+    - hacktoberfest
```

Comment:
I'm not sure how the labels are used, but should we have a `minifi` label instead, or in addition to, the `nifi` label?

Reply:
I didn't want to remove the `nifi` label as we already have that at the moment, but we can add the `minifi` label too.

Reply:
Added `minifi` label in 578ec141d168fe3ac0d84d3400b6c83dd4003313